include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -169,40 +170,40 @@ typedef void(ORT_API_CALL* OrtLoggingFunction)(
 /**
  * \param out Should be freed by `OrtReleaseEnv` after use
  */
-ORT_API_STATUS(OrtCreateEnv, OrtLoggingLevel default_warning_level, _In_ const char* logid, _Out_ OrtEnv** out)
+ORT_API_STATUS(OrtCreateEnv, OrtLoggingLevel default_logging_level, _In_ const char* logid, _Outptr_ OrtEnv** out)
 ORT_ALL_ARGS_NONNULL;
 
 /**
  * \param out Should be freed by `OrtReleaseEnv` after use
  */
 ORT_API_STATUS(OrtCreateEnvWithCustomLogger, OrtLoggingFunction logging_function,
-               _In_opt_ void* logger_param, OrtLoggingLevel default_warning_level,
+               _In_opt_ void* logger_param, OrtLoggingLevel default_logging_level,
                _In_ const char* logid,
-               _Out_ OrtEnv** out);
+               _Outptr_ OrtEnv** out);
 
 // TODO: document the path separator convention? '/' vs '\'
 // TODO: should specify the access characteristics of model_path. Is this read only during the
 // execution of OrtCreateSession, or does the OrtSession retain a handle to the file/directory
 // and continue to access throughout the OrtSession lifetime?
 //  What sort of access is needed to model_path : read or read/write?
 ORT_API_STATUS(OrtCreateSession, _In_ OrtEnv* env, _In_ const ORTCHAR_T* model_path,
-               _In_ const OrtSessionOptions* options, _Out_ OrtSession** out);
+               _In_ const OrtSessionOptions* options, _Outptr_ OrtSession** sess);
 
 ORT_API_STATUS(OrtCreateSessionFromArray, _In_ OrtEnv* env, _In_ const void* model_data, size_t model_data_length,
-               _In_ const OrtSessionOptions* options, _Out_ OrtSession** out);
+               _In_ const OrtSessionOptions* options, _Outptr_ OrtSession** sess);
 
-ORT_API_STATUS(OrtRun, _Inout_ OrtSession* sess,
+ORT_API_STATUS(OrtRun, _In_ OrtSession* sess,
```

Comment:
I guess here should be _Inout-, otherwise it should be const OrtSession* sess ?

Reply:
will fix